streams/src/main/java/org/apache/kafka/streams/processor/internals/PartitionGroup.java

```diff
@@ -41,8 +41,8 @@
  *
  * PartitionGroup also maintains a stream-time for the group as a whole.
```

Comment:
Maybe we should change "stream-time" to "task-time" ? @vvcephei suggested it (and I like it) in an in-person discussion? If we agree, also the corresponding variable and method names should be updated. Thoughts?

Should it be "stream-time" or "stream time" ?

Reply:
I support "task-time" -- would be good to distinguish from "stream time" should we ever want/need a "global" stream time.

Regarding "stream-time" vs "stream time" -- we use the hyphen when referring to types of time semantics (eg event-time) so I'd favor "stream time" to maintain a distinction between "semantics types" and "time definitions" -- WDYT?

Reply:
On the other hand, maybe phrases like "lowest task time" might be ambiguous (what does "lowest task" mean and why do we care about its time?) Not sure here